lib/build-utilities/migrate-initializers.js

```diff
@@ -89,10 +89,17 @@ function _moveFastBootInitializers(project, rootPath) {
  * @param {Object} project
  */
 function _migrateAddonInitializers(project) {
-  const nodeModulesPath = path.join(project.nodeModulesPath);
-
-  project.addons.forEach((addon) => {
-    const currentAddonPath = path.join(nodeModulesPath, addon.name);
+  // `nodeModulesPath` was removed in Ember 3, but we still support older
+  // versions, use empty string as a fallback
+  const legacyNodeModulesPath = path.join(project.nodeModulesPath || '');
+
+  project.addons.forEach(addon => {
+    const currentAddonPath = addon.root
```

Comment:
I think we can always rely on addon.root being the full path to the addons files. It has been around for a very long time (I think since nested addons were added back in 0.2.0).

Can you refactor to that alone (without fallback)?

Reply:
Sure thing

Reply:
@rwjblue all set 👍